### PR TITLE
android SSL support system certs

### PR DESCRIPTION
**Pull request by afpro**
Build with
cargo ndk -t arm64-v8a -p 23 build --release --features freeze-stdlib,ssl,ssl-vendor

Then push rustpython to /data/local/tmp/python

run /data/local/tmp/python with code:
import urllib.request
print(urllib.request.urlopen('https://httpbin.org/ip').read().decode('ascii'))

check if https connection successful

### CODE REVIEW

File: vm/src/stdlib/ssl.rs [hunk 1]
```diff
@@ -1257,3 +1260,83 @@ impl Write for SocketStream {
         socket.flush()
     }
 }
+
+#[cfg(target_os = "android")]
+mod droid {
```
Comment (youknowone) [inline vm/src/stdlib/ssl.rs:1265]: is there any reason this is not `android` but `droid`? are there more droids?

**Comment by youknowone** [comment]
patch to cleanup: https://github.com/RustPython/RustPython/pull/3163/commits/728af615ff96efb3403979b6b0384722d1e133ac

File: vm/src/stdlib/ssl.rs (hunk 1 above)
Comment (afpro) [reply vm/src/stdlib/ssl.rs:1265]: sorry about my bad naming style, i'll fix this soon.

**Review by youknowone — APPROVED** [review]

File: vm/src/stdlib/ssl.rs (hunk 1 above)
Comment (youknowone) [reply vm/src/stdlib/ssl.rs]: don't worry. I was actually wondered how it is named because I don't know about them well.

**Comment by youknowone** [comment]
Thank you for contributing!

**Pull request merged**